locales/fr_FR/LC_MESSAGES/duckduckgo.po

```diff
@@ -20,32 +20,35 @@ msgstr ""
 "X-Generator: Poedit 2.3\n"
 
 msgid "!Bang Search Shortcuts"
-msgstr "Raccourcis !Bang"
+msgstr "Raccourcis de recherche !Bang"
 
+#. input correct way to showcase amounts in French (done)
 msgctxt "showcase_donations"
 msgid "$800,000 in privacy donations!"
-msgstr "800,000 $ de donations privées !"
+msgstr "800 000 $ de dons privés !"
 
 msgctxt "SERP footer content"
 msgid "%s Billion Searches"
 msgstr "%s milliards de recherches"
 
 msgid "%s blocked by safe search."
-msgstr "%s bloqué par la recherche sécurisée."
+msgstr "%s bloqué par le module de recherche en toute sécurité."
 
 #. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
+#. fixed plurals
 msgctxt "published dates for organic results"
 msgid "%s days ago"
-msgstr "Il y a %s jours"
+msgstr "Il y a %s jour(s)"
 
 #. in https://duckduckgo.com/params. multiple fors
 msgid "%s for %s"
 msgstr "%s pour %s"
 
 #. We realise this is the incorrect way to handle plurals. Sorry, we are working on a fix.
+#. fixed plurals
 msgctxt "published dates for organic results"
 msgid "%s hours ago"
-msgstr "Il y a %s heures"
+msgstr "Il y a %s heure(s)"
```

Comment:
This is not needed. This string will always be plural. The singular version can be found below and it's already translated. (Also remove the comment at line 48 as explained above) Thanks!
```suggestion
```